attributes/master.rb

```diff
@@ -135,6 +135,11 @@
   master['use_system_accounts'] = true
 
   #
+  # The groups of user under which Jenkins is running. Works for runit only.
+  #
+  master['groups'] = [node['jenkins']['master']['group']]
```

Comment:
@mknapik Yes we always prefer a nice squashing. It's OK to have multiple commits but they should each stand on their own (with a nice commit message) that tell a story/narrative about the feature as a whole.